src/core/renderers/SystemRenderer.js

```diff
@@ -244,16 +244,17 @@ export default class SystemRenderer extends EventEmitter
      * @param {PIXI.DisplayObject} displayObject - The displayObject the object will be generated from
      * @param {number} scaleMode - Should be one of the scaleMode consts
      * @param {number} resolution - The resolution / device pixel ratio of the texture being generated
+	 * @param {PIXI.Rectangle} region - The region of the displayObject, that shall be rendered
```

Comment:
Documentation needs a little more love here. Should be `[region]` to denote that it's optional. Also, I would add a note that it default to the bounds of the displayObject. Otherwise 👍